### PR TITLE
Rewrites Trickwine Fluff

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
I tried my hand at rewriting the trickwine fluff to feel more... Lore. This included removing the overtly magical trickwines, rewriting the big fluff thing, and otherwise adjusting. 

## Why It's Good For The Game
Remains to be seen. Please leave your thoughts below. I will disregard them.

## Changelog

:cl:
add: Trickwine fluff has been rewritten
balance: Trickwines have been somewhat nerfed
del: prismwine, forcewine
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/trickwine_reagents.dm [hunk 1]
```diff
@@ -42,14 +38,16 @@
...
 	glass_name = "Icewine"
 	glass_desc = "A specialized brew utilized by members of the Saint-Roumain Militia, designed to assist in temperature regulation while working in hot environments. Known to give one the cold shoulder when thrown."
 	breakaway_flask_icon_state = "baflaskicewine"
 
 /datum/reagent/consumable/ethanol/trickwine/ice_wine/on_mob_life(mob/living/M)
 	M.adjust_bodytemperature(-5 * TEMPERATURE_DAMAGE_COEFFICIENT, M.get_body_temp_normal())
-	M.adjustFireLoss(-1)
+	if(prob(10))
+		to_chat(M, span_notice("Sweat runs down your body."))
+		M.adjustFireLoss(-0.5)
```
Comment (FalloutFalcon) [inline code/modules/reagents/chemistry/reagents/trickwine_reagents.dm]: Very strong nerf. Whats your reasoning?

File: code/modules/reagents/chemistry/reagents/trickwine_reagents.dm [hunk 2]
```diff
@@ -71,23 +69,23 @@
...
 
 /datum/reagent/consumable/ethanol/trickwine/shock_wine
-	name = "Shockwine"
+	name = "Lightning's Blessing"
 	description = "A stimulating brew utilized by members of the Saint-Roumain Militia, created to allow trackers to keep up with highly mobile prey. Known to have a shocking effect when thrown"
 	color = "#FEFEB8"
-	boozepwr = 70
-	taste_description = "the adrenaline of the chase"
-	glass_name = "Shockwine"
+	boozepwr = 50
```
Comment (FalloutFalcon) [inline code/modules/reagents/chemistry/reagents/trickwine_reagents.dm:75]: Booze power is a balancing element

**Review by FalloutFalcon — CHANGES_REQUESTED** [review]
I have more thoughts, a further review later
Whats your reasoning for removing prism

File: code/modules/reagents/chemistry/reagents/trickwine_reagents.dm (hunk 2 above)
Comment (Erikafox) [reply code/modules/reagents/chemistry/reagents/trickwine_reagents.dm:75]: I am aware.

File: code/modules/reagents/chemistry/reagents/trickwine_reagents.dm (hunk 1 above)
Comment (Erikafox) [reply code/modules/reagents/chemistry/reagents/trickwine_reagents.dm]: I wanted to move it away from being on par with dedicated healing chemicals, while retaining the use it had

**Comment by Erikafox** [comment]
> I have more thoughts, a further review later Whats your reasoning for removing prism

I put some thoughts into it and couldn't come to a base that I was satisfied with

File: code/modules/reagents/chemistry/reagents/trickwine_reagents.dm [hunk 3]
```diff
@@ -3,26 +3,22 @@
...
-		to_chat(M, "<span class='notice'>[high_message]</span>")
-	if(M.faction && ("roumain" in M.faction))
-		M.adjustToxLoss(-2)
-		if(prob(10))
-			to_chat(M, "<span class='notice'>[cleanse_message]</span>")
+	if(prob(15))
+		M.adjustToxLoss(-1)
+		M.adjust_drugginess(5)
+		var/high_message = pick("Devotion runs wild within your soul", "A lust for hunting leaps from within your psyche", "The inner beauty of nature courses within your minds' eye.", "Calm warmth spreads within your body.")
+		to_chat(M, span_notice("[high_message]"))
```
Comment (FalloutFalcon) [inline code/modules/reagents/chemistry/reagents/trickwine_reagents.dm:21]: The idea for ash wine was that it was a minor perk for the srm to be able to clear toxin with it
there bodies have adapted to the bacteria and are able to utilize it better

File: code/modules/reagents/chemistry/reagents/trickwine_reagents.dm (hunk 1 above)
Comment (FalloutFalcon) [reply code/modules/reagents/chemistry/reagents/trickwine_reagents.dm]: The issue is that at present that effect is easily the weakest of all the wine effects, Im fine with tuning down the healing but 20x as worse seems extreame

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 4]
```diff
@@ -633,22 +633,10 @@
 
 /datum/chemical_reaction/hearth_wine
 	results = list(/datum/reagent/consumable/ethanol/trickwine/hearth_wine = 5)
-	required_reagents = list(/datum/reagent/consumable/ethanol/hcider = 3, /datum/reagent/consumable/pyre_elementum = 1, /datum/reagent/fuel = 1)
+	required_reagents = list(/datum/reagent/consumable/ethanol/hcider = 3, /datum/reagent/consumable/pyre_elementum = 1, /datum/reagent/phosphorus = 1)
```
Comment (FalloutFalcon) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm:636]: phosphorus is a fair bit harder to obtain and it already has a plant regent with prye elementum and its alot easier to make it with ghetto chem rather then growing glowshrooms. hearthwine is already pretty rare to made compared to other wines.
The scheme I have for trickwine recipes is (Booze, Plant regeant, Some other chemical who's best source is a mob or other biome specific feature)

File: code/modules/reagents/chemistry/reagents/trickwine_reagents.dm [hunk 5]
```diff
@@ -3,26 +3,22 @@
...
-	name = "Ashwine"
-	description = "A traditional sacrament for members of the Saint-Roumain Militia. Known to grant visions, and is used both for ritual and entertainment purposes aboard Saint-Roumain vessels."
+	name = "Wine Of Ash"
+	description = "A traditional sacrament for members of the Saint-Roumain Militia. Believed to grant visions, seeing use both in ritual and entertainment within the Militia."
 	color = "#6CC66C"
 	boozepwr = 80
 	quality = DRINK_VERYGOOD
-	taste_description = "devotional energy and a hint of high-potency hallucinogens"
+	taste_description = "a rustic fruit, with hints of sweet yet tangy ash."
 	glass_name = "Ashwine"
```
Comment (FalloutFalcon) [inline code/modules/reagents/chemistry/reagents/trickwine_reagents.dm]: Not sold on the name change but still,
```suggestion
	glass_name = "Wine of Ash"
```

File: code/modules/reagents/chemistry/reagents/trickwine_reagents.dm [hunk 6]
```diff
@@ -42,14 +38,16 @@
 	description = "A specialized brew utilized by members of the Saint-Roumain Militia, designed to assist in temperature regulation while working in hot environments. Known to give one the cold shoulder when thrown."
 	color = "#C0F1EE"
 	boozepwr = 70
-	taste_description = "a cold night on the hunt"
+	taste_description = "a weighty meat, undercut by a mild pepper."
 	glass_name = "Icewine"
```
Comment (FalloutFalcon) [inline code/modules/reagents/chemistry/reagents/trickwine_reagents.dm]:
```suggestion
	glass_name = "Wine of Ice"
```

Comment (FalloutFalcon) [reply code/modules/reagents/chemistry/reagents/trickwine_reagents.dm]: and the name

**Review by FalloutFalcon — DISMISSED** [review]
Also you will want to get a review from valorium on the book but I wouldn't call them alchemists as I would prefer using brewing related terminology to lean away from them just being ghetto chemists.

File: code/modules/reagents/chemistry/reagents/trickwine_reagents.dm (hunk 3 above)
Comment (Erikafox) [reply code/modules/reagents/chemistry/reagents/trickwine_reagents.dm:21]: That can potentially be tied in as a trait at a later date. Checking for the mobs faction is... weird.

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm (hunk 4 above)
Comment (Erikafox) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm:636]: Admittedly. I think its more interesting to get glowshrooms or grind down matches when the alternative is fuel of the weld

File: code/modules/reagents/chemistry/reagents/trickwine_reagents.dm (hunk 3 above)
Comment (FalloutFalcon) [reply code/modules/reagents/chemistry/reagents/trickwine_reagents.dm:21]: its fine to use mob faction..

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm (hunk 4 above)
Comment (FalloutFalcon) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm:636]: My issue is its WAY easier to get a decent amount of phosphorus via ghetto chem then both of those alternatives.

**Pull request merged**